Robus/src/robus.c

```diff
@@ -231,6 +231,13 @@ uint16_t Robus_TopologyDetection(ll_service_t *ll_service)
     uint8_t redetect_nb = 0;
     bool detect_enabled = true;
 
+    // if a detection is in progress,
+    // Don't do an another detection and return 0
+    if (Robus_IsNodeDetected() == NETWORK_LINK_CONNECT)
+    {
+        return 0;
+    }
+
```

Comment:
@ncros3 if you have time check a bit @nicolas-rabault comments regarding end detection

Reply:
I think this check has to be done as soon as possible: if a node wants to launch a detection but it cannot due to a prior detection in progress, it must have this information early and stop the process.

Reply:
@ncros3 Agreed

Reply:
Yes indeed, but if you receive the information of an occurring detection just after this if statement this will result in a  failed detection because you will launch concurrent detections without knowing that another one is occurring.
So you should check it just before the transmission of the detection start (or even better, just after) to limit detection collisions.
Alternatively, if we need it we could make this check twice.

Reply:
Ok, I get it. I added a check before the call to Robus_SendMsg().